pkg/networkverifier/networkverifier.go

```diff
@@ -97,8 +96,13 @@ func Run(cluster *v1.Cluster, clusterDeployment *hivev1.ClusterDeployment, awsCl
 		return Undefined, "", fmt.Errorf("failed to initialize validateEgressInput: %w", err)
 	}
 
-	credentials := awsClient.GetAWSCredentials()
-	awsVerifier, err := verifierAws.NewAwsVerifier(credentials.AccessKeyID, credentials.SecretAccessKey, credentials.SessionToken, cluster.Region().ID(), "", false)
+	// The network verifier requires a very specific type of logger (ocm logger)
+	logger, err := ocmlog.NewStdLoggerBuilder().Build()
```

Comment:
This might be worth a followup: we're not able to use our structured logger because of this.

Reply:
This is on our radar, probably a roadmap improvement alongside passing a context from the caller